test/units/testsuite-74.bootctl.sh

```diff
@@ -60,7 +60,13 @@ basic_tests() {
 
 testcase_bootctl_basic() {
     assert_eq "$(bootctl --print-esp-path)" "/efi"
-    assert_eq "$(bootctl --print-boot-path)" "/boot"
+    case "$(bootctl --print-boot-path)" in
+        /boot|/efi)
+            ;;
+        *)
+            assert_not_reached
+            ;;
+    esac
```

Comment:
The esp path can end up on /boot as well so please apply the same switch to both commands.